nfd-operator/base/nodefeaturediscoveries/nfd-instance.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     image: registry.redhat.io/openshift4/ose-node-feature-discovery:v4.11
     imagePullPolicy: Always
     servicePort: 0
-  topologyUpdater: false
+  topologyupdater: false
```

Comment:
Your commit message says "`topologyUpdated` should be spelled `topologyupdated`" but here it's ending in "er" (Updater -> updater). Which is it?

Reply:
My commit message is wrong. Fixing it...